tree/ntuple/v7/test/ntuple_merger.cxx

```diff
@@ -831,6 +831,36 @@ static bool VerifyValidZLIB(const void *buf, size_t bufsize, size_t tgtsize)
    return true;
 }
 
+enum class PageCompCheckType { kUncompressed, kZlib };
+
+static bool VerifyPageCompression(const std::string_view fileName, PageCompCheckType checkType)
+{
+   // TODO(gparolini): eventually we want to do the following check:
+   //   auto reader = RNTupleReader::Open("ntuple", fileGuardOut.GetPath());
+   //   auto compSettings = reader->GetDescriptor().GetClusterDescriptor(0).GetColumnRange(0).fCompressionSettings;
+   //   EXPECT_EQ(compSettings, kNewComp);
+   // but right now we don't write the correct metadata when calling Merge() so we can't trust the advertised
```

Comment:
Actually even when that is fixed, we may want to test here both ways to make sure they are consistent.